lms/envs/common.py

```diff
@@ -2323,6 +2323,7 @@
 }
 
 # E-Commerce API Configuration
+ECOMMERCE_EXTERNAL_URL_ROOT = None
```

Comment:
Thanks for the suggestion, guys. I've opted for `ECOMMERCE_PUBLIC_URL_ROOT` to reflect the fact that the setting is meant to contain a URL root, not just any URL.